modules/extdata/platform/pom.xml

```diff
@@ -42,5 +42,13 @@
             <version>${project.version}</version>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.apache.ignite</groupId>
+            <artifactId>ignite-core</artifactId>
```

Comment:
Why dependency duplicated?

Reply:
There is no duplication. The first dependency for ignite-core production classes, the second for corresponding test classes.